code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -218,3 +218,24 @@
 	icon = 'icons/effects/blood.dmi'
 	icon_state = "xfloor1"
 	random_icon_states = list("xfloor1", "xfloor2", "xfloor3", "xfloor4", "xfloor5", "xfloor6", "xfloor7")
+
+//monkestation edit begin
```

Comment:
This section could be modular

Reply:
Done!  decals/cleanable/misc.dm should now be untouched.  The decals are now in the monkestation folder.